### DISCUSSION TITLE
How to I Add Mods to It

### DISCUSSION THREAD

**Discussion by TouseefX** [Impostor/Impostor#555] [Q&A]
I Don't Know to add Mods into It, I AM Using Among US Itch.io Version, but there is No Mods for Itch.io version the mod can be installed to itch.io the reason but is harder and there are any way that we add mod in this

**Comment by miniduikboot** [comment]
Hi there,

You can just install mods like Town of Us or TheOtherRoles in same way as for the Steam or Epic versions of the game, the Itch version is usually just not listed because there aren't many people using it. If you need to downgrade to an older version you can do so using the Itch desktop client.

However Impostor is sort of a special: it's not a client mod, but rather a replacement for the official servers. If you need support for client mods, you're better off asking the mod authors directly, usually via Discord. If you need a server to play your modded game on, try Impostor ;)

**Reply by TouseefX** [reply]
ok but nvm i foudn way to do it

**Reply by TouseefX** [reply]
https://amodsus.com/resources/impolobbycommands.95/

**Reply by TouseefX** [reply]
Plugin (Impostor server) https://amodsus.com/resources/categories/plugin-impostor-server.2/

**Reply by miniduikboot** [reply]
That's a plugin you can install into your server. But as it is quite old, it may no longer work (completely)

**Reply by TouseefX** [reply]
is working tho
